File: SWEETMANAGER/LARAVEL/app/Models/Instituicao.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Instituicao extends Model
{
    use HasFactory;

    protected $table = 'instituicoes';

    protected $fillable = ['nome', 'contato', 'cnpj'];

    public function vendas()
    {
        return $this->hasMany(Venda::class);
    }
}

File: SWEETMANAGER/FLUTTER/lib/models/instituicao.dart
class Instituicao {
  final int id;
  final String nome;
  final String contato;
  final String cnpj;

  Instituicao({
    required this.id,
    required this.nome,
    required this.contato,
    required this.cnpj,
  });

  factory Instituicao.fromJson(Map<String, dynamic> json) {
    return Instituicao(
      id: json['id'],
      nome: json['nome'],
      contato: json['contato'],
      cnpj: json['cnpj'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'nome': nome,
      'contato': contato,
      'cnpj': cnpj,
    };
  }
}
